test/bdd/features/redeemable_routes.feature

```diff
@@ -1,24 +0,0 @@
-#
```

Comment:
what happened to this file?

Reply:
See https://github.com/hyperledger/aries-framework-go/pull/2729#discussion_r614048232